sub-wheels/matplotlib-baseline-images/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+import os
+from pathlib import Path
+import sys
+import subprocess
+
+__version__ = subprocess.check_output(
```

Comment:
perhaps add a short comment explaining why we need such a roundabout way to get the version.